analytics/config.py

```diff
@@ -20,7 +20,9 @@
     validators=[
         Validator("SLACK_BOT_TOKEN", must_exist=True),
         Validator("REPORTING_CHANNEL_ID", must_exist=True),
-    ],
+    ]
+    database_url="postgresql://user:password@location:port/dbname" # get vars from the api?
+    ,
```

Comment:
If you've got access to env vars here, could build this like we do for the flask API:

https://github.com/navapbc/simpler-grants-gov/blob/main/api/src/adapters/db/clients/postgres_client.py#L79

This just takes in a config object that is loading env vars, defined in: https://github.com/navapbc/simpler-grants-gov/blob/main/api/src/adapters/db/clients/postgres_config.py

Handles IAM auth as well.

---

Note, you might want the start of this to be `postgresql+psycopg` to tell it to use the latest version of psycopg (I think it defaults to psycopg2? They named the 3rd version just psycopg, its weird): https://docs.sqlalchemy.org/en/20/dialects/postgresql.html#module-sqlalchemy.dialects.postgresql.psycopg